### ISSUE TITLE
tensorflow v1.8 rocm version failed  tf/models/torurial/rnn/ptb  samples under medium level model.

### ISSUE THREAD

**Issue by JiniusDnn** [ROCmSoftwarePlatform/tensorflow-upstream#121]
Here is the log:

:~/tf/models/tutorials/rnn/ptb$ python ptb_word_lm.py --data_path=./simple-examples/data/ --model=medium
2018-08-14 16:29:27.761637: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-08-14 16:29:27.771146: W tensorflow/stream_executor/rocm/rocm_driver.cc:404] creating context when one is currently active; existing: 0x7f5ca11bf930
2018-08-14 16:29:27.771323: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1451] Found device 0 with properties: 
name: Device 6863
AMDGPU ISA: gfx900
memoryClockRate (GHz) 1.6
pciBusID 0000:03:00.0
Total memory: 15.98GiB
Free memory: 15.73GiB
2018-08-14 16:29:27.771334: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1562] Adding visible gpu devices: 0
2018-08-14 16:29:27.771342: I tensorflow/core/common_runtime/gpu/gpu_device.cc:989] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-08-14 16:29:27.771346: I tensorflow/core/common_runtime/gpu/gpu_device.cc:995]      0 
2018-08-14 16:29:27.771349: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1008] 0:   N 
2018-08-14 16:29:27.771381: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1124] Created TensorFlow device (/device:GPU:0 with 15306 MB memory) -> physical GPU (device: 0, name: Device 6863, pci bus id: 0000:03:00.0)
WARNING:tensorflow:From ptb_word_lm.py:504: Supervisor.__init__ (from tensorflow.python.training.supervisor) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.MonitoredTrainingSession
2018-08-14 16:29:35.098606: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1562] Adding visible gpu devices: 0
2018-08-14 16:29:35.098664: I tensorflow/core/common_runtime/gpu/gpu_device.cc:989] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-08-14 16:29:35.098672: I tensorflow/core/common_runtime/gpu/gpu_device.cc:995]      0 
2018-08-14 16:29:35.098678: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1008] 0:   N 
2018-08-14 16:29:35.098727: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1124] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 15306 MB memory) -> physical GPU (device: 0, name: Device 6863, pci bus id: 0000:03:00.0)
WARNING:tensorflow:Standard services need a 'logdir' passed to the SessionManager
Epoch: 1 Learning rate: 1.000
0.008 perplexity: 4897.639 speed: 2075 wps
0.107 perplexity: 1183.778 speed: 2219 wps
0.206 perplexity: 859.641 speed: 2229 wps
0.306 perplexity: 691.848 speed: 2231 wps
0.405 perplexity: 595.160 speed: 2233 wps
Memory access fault by GPU node-1 (Agent handle: 0x2bb6560) on address 0xffe000. Reason: Page not present or supervisor privilege.

**Comment by whchung** [comment]
In order to help pin down the faulty GPU kernel, please help run your model with the following environment variables:
```
HCC_SERIALIZE_KERNEL=3 HCC_SERIALIZE_COPY=3 HIP_TRACE_API=2 MIOPEN_ENABLE_LOGGING=1
```

**Comment by JiniusDnn** [comment]
I can't re-create this crash after refresh boot.  Feel it's somewhere G-ram polluted cause this issue. Will reopen it if I am able to see it again.
